src/main/webapp/app/faq/faq-update.component.ts

```
@@ -31,6 +32,7 @@
     existingCategories: FaqCategory[];
     faqCategories: FaqCategory[];
     courseId: number;
+    isAtleastInstructor: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)